test/y2network/interface_config_builders/bonding_test.rb

```diff
@@ -55,6 +55,10 @@
       allow(connection_config).to receive(:name).and_return(connection_name)
 
       allow(connection_config).to receive(:find_master).and_return(connection_master)
+
+      allow(Yast::FileUtils).to receive(:IsDirectory).and_return(true)
```

Comment:
wasn't better to call the s390 method calls?

Reply:
maybe we could move that check of layer2 support to a convenience method and mock only it

Reply:
Maybe, but I am not sure if it is worth doing it now. That changes would no be related with the fix about Yast namespace.